package.json

```diff
@@ -2,83 +2,79 @@
-    "ember-autosave": "2.0.1",
-    "ember-bootstrap": "0.7.2",
-    "ember-browserify": "1.1.13",
-    "ember-cli": "2.11.1",
-    "ember-cli-app-version": "1.0.0",
-    "ember-cli-babel": "5.1.6",
-    "ember-cli-bootstrap-sassy": "0.5.3",
-    "ember-cli-dependency-checker": "1.2.0",
-    "ember-cli-dropzonejs": "0.7.0",
-    "ember-cli-eslint": "3.0.3",
-    "ember-cli-htmlbars": "1.0.3",
-    "ember-cli-htmlbars-inline-precompile": "0.3.6",
-    "ember-cli-inject-live-reload": "1.4.0",
-    "ember-cli-inline-content": "0.4.0",
-    "ember-cli-meta-tags": "3.0.1",
-    "ember-cli-postcss": "3.0.1",
-    "ember-cli-qunit": "2.0.0",
-    "ember-cli-release": "0.2.9",
-    "ember-cli-sass": "5.3.1",
-    "ember-cli-sri": "2.1.0",
-    "ember-cli-template-lint": "0.4.12",
-    "ember-cli-test-loader": "1.1.0",
-    "ember-cli-uglify": "1.2.0",
-    "ember-collapsible-panel": "1.0.0",
-    "ember-collection": "1.0.0-alpha.6",
-    "ember-cp-validations": "2.9.4",
-    "ember-data": "2.7.0",
-    "ember-export-application-global": "1.0.5",
-    "ember-font-awesome": "2.1.1",
-    "ember-get-config": "0.0.4",
-    "ember-i18n": "4.3.1",
-    "ember-link-action": "0.0.35",
-    "ember-load-initializers": "0.5.1",
-    "ember-metrics": "0.6.3",
-    "ember-page-title": "3.0.6",
-    "ember-power-select": "1.0.0-beta.7",
-    "ember-read-more": "0.0.13",
-    "ember-resolver": "2.1.0",
-    "ember-sortable": "1.8.2",
-    "ember-toastr": "1.4.1",
-    "ember-truth-helpers": "1.2.0",
-    "ember-welcome-page": "1.0.1",
-    "eslint": "3.18.0",
-    "liquid-fire": "0.24.1",
-    "loader.js": "4.0.1",
-    "pagination-pager": "2.4.2",
-    "postcss": "5.1.0"
+    "broccoli-asset-rev": "^2.4.5",
+    "ember-bootstrap": "^1.0.0-rc.1",
+    "ember-browserify": "^1.2.0",
+    "ember-cli": "2.18.0",
+    "ember-cli-app-version": "^3.0.0",
+    "ember-cli-babel": "^6.7.1",
+    "ember-cli-bootstrap-sassy": "0.5.6",
+    "ember-cli-dependency-checker": "^2.0.1",
+    "ember-cli-eslint": "^4.2.0",
+    "ember-cli-htmlbars": "^2.0.1",
+    "ember-cli-htmlbars-inline-precompile": "^1.0.2",
+    "ember-cli-inject-live-reload": "^1.4.1",
+    "ember-cli-inline-content": "^0.4.1",
+    "ember-cli-meta-tags": "4.0.0",
+    "ember-cli-moment-shim": "^3.4.0",
+    "ember-cli-postcss": "^3.5.2",
+    "ember-cli-qunit": "^4.0.0",
+    "ember-cli-sass": "^7.0.0",
+    "ember-cli-shims": "^1.1.0",
+    "ember-cli-sri": "^2.1.0",
+    "ember-cli-template-lint": "^0.7.5",
+    "ember-cli-uglify": "^1.2.0",
+    "ember-data": "2.18.0",
+    "ember-export-application-global": "^2.0.0",
+    "ember-font-awesome": "^3.0.0",
+    "ember-get-config": "^0.2.4",
+    "ember-i18n": "5.0.2",
+    "ember-link-action": "^0.0.36",
+    "ember-load-initializers": "^1.0.0",
+    "ember-moment": "^7.4.1",
+    "ember-page-title": "^3.2.2",
+    "ember-resolver": "^4.0.0",
+    "ember-source": "2.18.0",
+    "ember-toastr": "^1.7.0",
+    "eslint": "^4.4.1",
+    "hint.css": "^2.5.0",
```

Comment:
This may need to change, see licensing note on ember-osf-preprints side:

https://github.com/CenterForOpenScience/ember-osf-preprints/blob/develop/bower.json#L11

Didn't look hard enough to know why, but it is worth looking into.

Reply:
Looks like you're correct. Versions after `2.3.2` are still free for personal use, but since we `are using the library for business, commercial sites, themes, projects, and applications, the Commercial license is the appropriate license.`

We could reach out and see if they make exceptions for non-profits.

Reply:
I did a quick look through and didn't see anywhere specific that we're using `hint.css` so I'm going to try to remove it.  
If we need to use it in the future, we can always go back to using version `2.3.2`.